src/app/home/home.component.ts

```diff
@@ -15,13 +15,21 @@ import { Fabric8UIConfig } from '../shared/config/fabric8-ui-config';
 import fabric8Logo from '../../assets/images/fabric8_logo.png';
 import openshiftLogo from '../../assets/images/OpenShift-io_logo.png';
 
+import { FeatureTogglesService } from '../feature-flag/service/feature-toggles.service';
+
 @Component({
   encapsulation: ViewEncapsulation.None,
   selector: 'alm-home',
   templateUrl: './home.component.html',
   styleUrls: ['./home.component.less']
 })
 export class HomeComponent implements OnInit, OnDestroy {
+  newHomeDashboardEnabled: boolean = false;
+  developmentEnabled: boolean = false; // set to false to hide for prod - set to true for local development
+  myInterval: number = 0;
+  noWrapSlides: boolean = true;
```

Comment:
We can probably set the myInterval and noWrapSlides properties on the carousel directly, in the HTML template, as these are not likely to change.

Reply:
I'm using the methods documented on the ngx-bootstrap site: https://valor-software.com/ngx-bootstrap/#/carousel. They have it broken out into template and component, even for a single item.

Reply:
Yes, nothing wrong with following the example -- no biggie. However, it can be simplified to just this:

<carousel interval="0" noWrap="true">